.nvmrc

```diff
@@ -1 +1 @@
-10.16.3
+v12.22
```

Comment:
no "v" prefix in a .nvmrc 🙏

Reply:
Hmm, it works on my machine with the "v". Also when I nvm list, I see vXXXX. Let me deep in to see why

Reply:
https://github.com/nvm-sh/nvm#nvmrc

$ echo "5.9" > .nvmrc

no "v" in the official documentation. Even if I'm not sure this is a game changer.